package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Pin mapping functionality for the Raspberry Pi in Node.js",
   "main": "lib/index.js",
   "scripts": {
+    "build": "gulp",
```

Comment:
I have considered that setup. It's something I want to do, but I think it's necessary to get proper unit tests written first that are also added to prepublish.

Currently, when I do a release, I first run a number of manual tests as a sanity check to make sure nothing is broken. It's not ideal, but it's at least something. I have a personal requirement that whatever gets published must be tested first, and no changes to files of any sort are allowed between testing and publishing.

If we set the build as a prepublish step with the current infrastructure, it then becomes possible for what is tested to be different from what is published, which increases the chance of bugs.

Once we have proper unit tests though, then that removes the chance of this bug occurring, and we can add it as prebublish. I have a plan to get unit tests implemented, and I'm waiting for a board I pre-ordered to arrive that will make this a lot easier, so stay tuned!